CONTRIBUTING.md

```diff
@@ -125,6 +126,32 @@ See [the API reference](https://docs.llamaindex.ai/en/latest/api_reference/index
 
 ---
 
+
+
+
+### Difference between AutoMerging, SentenceWindow and others, with respect to indexing, query engines and node parsers.
+
+Let's break down the differences between AutoMerging, SentenceWindow, and generic node parsers like sentence splitters:
+
+1. **AutoMerging**:
+   - **Functionality**: AutoMerging is primarily concerned with identifying and merging similar or related documents or data entities to reduce redundancy and improve efficiency. It involves algorithms that analyze the content of documents or data sets and automatically group together items that share common characteristics.
+   - **Purpose**: The main purpose of AutoMerging is to streamline the search process and improve the organization and presentation of search results by eliminating duplicate or highly similar entries.
+   - **Example**: In a search engine, AutoMerging might involve aggregating search results from different sources or indexes that contain similar content into a single unified result set.
+
+2. **SentenceWindow**:
+   - **Functionality**: SentenceWindow techniques focus on considering the context or surrounding sentences of a given sentence or query term. This is often used to capture the semantic context of the text, which can aid in tasks such as document summarization, context-aware search, or entity recognition.
+   - **Purpose**: The purpose of SentenceWindow techniques is to provide a more nuanced understanding of the text by considering the surrounding context, which can lead to more accurate parsing, analysis, and retrieval of information.
```

Comment:
It uses a single sentence for retrieval, and then the surrounding window after retrieval when sending to the LLM for response synthesis